Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -1068,4 +1068,131 @@ function Set-ClusterDefaultStoragePolicy {
     }
 }
 
-Export-ModuleMember -Function *
+<#
+    .Synopsis
+    Scale the HCX manager vm to the new resource allocation of 8 vCPU and 24 GB RAM (Default 4 vCPU/12GB)
+#>
+function Set-HcxScaledCpuAndMemorySetting {
+    try {
+        $DefaultViConnection = $DefaultVIServers
+
+        Write-Host "Creating new temp scripting user"
+        $HcxAdminCredential = New-TempUser -privileges @("VirtualMachine.Config.CPUCount","VirtualMachine.Config.Memory") -userName tempHcxAdmin -userRole tempHcxAdminRole
+        Write-Host "User created"
+
+        $Port = 443
+        $HcxServer = 'hcx'
+        $HcxPreferredVersion = '4.3.2'
+        $SlashCommonTreshold = '90%'
+        $HcxScaledtNumCpu = 8
+        $HcxScaledMemoryGb = 24
+        $Found = $false
+
+        Write-Host "Identifying HCX Vm"
+        $VmsList = Get-VM
+        foreach ($Vm in $VmsList) {
```

Comment:
Where is the connection with the new user?

Reply:
Line 1084. Post the user creation.